src/Commands/ShifterToggle.cpp

```diff
@@ -0,0 +1,61 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+
+#include "ShifterToggle.h"
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+ShifterToggle::ShifterToggle(): Command() {
+        // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+	Requires(Robot::chassis.get());
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+}
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+// Called just before this Command runs the first time
+void ShifterToggle::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void ShifterToggle::Execute() {
+	//High Speed Transmission
+	if(RobotMap::chassisshiftLeft->Get() == DoubleSolenoid::kForward)
+	{
+		RobotMap::chassisshiftLeft->Set(DoubleSolenoid::kReverse);
```

Comment:
We can work with RobotBuilder and avoid naming anything with a begging lowercase letter and continue using CamelCase from then on. Such as ShiftLeft in Chassis which would be converted to chassisShiftLeft by RobotBuilder, so you would have to change the name in RobotBuilder @minermarty to fix that issue